packages/shared/src/lib/featureManagement.ts

```diff
@@ -30,6 +30,9 @@ const feature = {
   onboardingChecklist: new Feature('onboarding_checklist', true),
   searchUsers: new Feature('search_users', false),
   showCodeSnippets: new Feature('show_code_snippets', false),
+  searchText: new Feature('search_text', {
+    placeholder: 'Search',
+  }),
```

Comment:
Normally, we resort to using just a string when it comes to copy changes so this can be just a string literal.
```suggestion
  searchText: new Feature('search_text', 'Search'),
```

Reply:
@sshanzel I created that just in case product wants to play with different parts of search text. But we can convert if we want to I don't mind.

Reply:
Changed to string only since it doesn't seem product want to experiment differently 

https://app.growthbook.io/features/search_placeholder

flag is search_placeholder and is a string defaulting to Search

cc. @AmarTrebinjac